packages/mdc-snackbar/_variables.scss

```diff
@@ -20,9 +20,37 @@
 // THE SOFTWARE.
 //
 
-// Hard coded since the color is not present in any palette.
-$mdc-snackbar-background-color: #323232 !default;
-$mdc-snackbar-foreground-color: white !default;
-// TODO: Better spot to pull this breakpoint?
-//$snackbar-tablet-breakpoint: $grid-tablet-breakpoint;
-$mdc-snackbar-tablet-breakpoint: 600px !default;
+@import "@material/theme/functions";
+@import "@material/theme/variables"; // for mdc-theme-prop-value() function
+@import "@material/theme/color-palette";
+
+//
+// Public
+//
+
+$mdc-snackbar-surface-fill-color: surface !default;
+$mdc-snackbar-overlay-fill-color: on-surface !default;
+$mdc-snackbar-label-ink-color: surface !default;
+$mdc-snackbar-action-button-ink-color: #bb86fc !default;
+$mdc-snackbar-action-icon-ink-color: surface !default;
+
+$mdc-snackbar-label-type-scale: body2 !default;
+$mdc-snackbar-action-icon-size: 18px !default;
+$mdc-snackbar-min-width: 344px !default;
+$mdc-snackbar-max-width: 672px !default;
+$mdc-snackbar-viewport-margin-narrow: 8px !default;
+$mdc-snackbar-viewport-margin-wide: 24px !default;
+$mdc-snackbar-padding: 8px !default;
+
+$mdc-snackbar-elevation: 6 !default;
+$mdc-snackbar-shape-radius: small !default; // Key from $mdc-shape-category-values or CSS length value (e.g., 4px)
+$mdc-snackbar-z-index: 8 !default; // One above mdc-dialog
+
+//
+// Private
+//
+
+$mdc-snackbar-enter-delay_: 0ms;
```

Comment:
Is there a reason these are distinguished as private, given the request we've gotten to expose everything as overrideable?

Reply:
Good call—made them "public"